deploy/your-repo-name/values/staging.yaml

```diff
@@ -50,12 +50,12 @@ service:
 ingress:
   enabled: true
   hosts:
-    - host: your-name-staging.cloud-platform.service.justice.gov.uk
+    - host: your-staging-namespace.cloud-platform.service.justice.gov.uk
       paths:
         - path: /
           pathType: Prefix
   annotations:
-    external-dns.alpha.kubernetes.io/set-identifier: "ingress-name-your-repo-name-your-repo-name-staging-green"
+    external-dns.alpha.kubernetes.io/set-identifier: "dns-identifier-for-staging"
```

Comment:
I prefer this pattern:  'external-dns.alpha.kubernetes.io/set-identifier: "ingress-name-your-repo-name-your-repo-name-staging-green"'